src/jwt/__tests__/jwt.spec.js

```diff
@@ -114,30 +115,15 @@ describe('id token verification tests', () => {
     it('passes verification with valid token signed with RS256', async () => {
       const testJwt =
         'eyJhbGciOiJSUzI1NiIsImtpZCI6IjEyMzQifQ.eyJpc3MiOiJodHRwczovL3Rva2Vucy10ZXN0LmF1dGgwLmNvbS8iLCJzdWIiOiJhdXRoMHwxMjM0NTY3ODkiLCJhdWQiOlsidG9rZW5zLXRlc3QtMTIzIiwiZXh0ZXJuYWwtdGVzdC0xMjMiXSwiZXhwIjoxNTY3NDg2ODAwLCJpYXQiOjE1NjczMTQwMDAsIm5vbmNlIjoiYTU5dms1OTIiLCJhenAiOiJ0b2tlbnMtdGVzdC0xMjMiLCJhdXRoX3RpbWUiOjE1NjczMTQwMDB9.ObH7oG3NsGaxWnB8rzbLOgAD2I0fr9dyZC81YUrbju3RwC3lRAxqJkbesiSdGKry9OamIhKYwUGpPK0wrBaRJo8UjDjICkhM6lGP23plysemxhDnFK1qjj-NaUaW1yKg14v2lVpQl7glW9LIhFDhpqIf4bILA2wt9-z8Uvi31ETZvGb8PDY2bEvjXR-69-yLuoTNT2skP9loKfz6hHDMQCTWrGA61BMMjkZBLo9UotD9BzN8V7bLrFFT25v6q9N83mWaGLsHntzPIl3EYPOwX0NbE0lXKar59TUqtaTB3uNFHbGjIYi8wuuIp4PV9arpE3YrjWOOmrMurD1KpIyQrQ';
-      const contents = fs.readFileSync(
-        path.resolve(__dirname, './pubkey.pem'),
-        'utf8',
-      );
 
-      const pubKey = KEYUTIL.getKey(contents);
-      const jwkFromKey = KEYUTIL.getJWKFromKey(pubKey);
-
-      jwkFromKey.kid = '1234';
-      jwkFromKey.alg = 'RS256';
-      jwkFromKey.use = 'sig';
-
-      const jwks = {
-        keys: [jwkFromKey],
-      };
-
-      setupFetchMock({jwks});
+      setupFetchMock();
 
       await expect(verify(testJwt)).resolves.toBeUndefined();
     });
 
     const setupFetchMock = ({
       domain = BASE_EXPECTATIONS.domain,
```

Comment:
Despite hoisting, shouldn't this be put at the top, or at the bottom (or in a helpers file), instead of being in the middle of the file?

Reply:
I'll see if I can move it upwards 👍 thanks

Reply:
Moved to the bottom